src/containers/DataDisplay/DataViewTable.tsx

```diff
@@ -1,86 +1,176 @@
+              open={false}
+              value={searchText}
+            />
+          )}
+        </div>
+        <div className="flex-row-space-between-center" style={{ width: '100%' }}>
+          <ListFilters filterDefinitions={filterDefinitions} filterListParameters={{}} />
+        </div>
+        {loading && <Loading />}
+        {dataViewTable && (
+          <DataViewTableContent dataViewTable={dataViewTable} apiQueries={apiQueries} />
+        )}
       </div>
     </div>
   )
 }
 
 export default DataViewTable
 
+interface DataViewTableContentProps {
+  dataViewTable: DataViewsTableResponse
+  apiQueries: DataViewTableAPIQueries
+}
+
+const DataViewTableContent: React.FC<DataViewTableContentProps> = ({
+  dataViewTable,
+  apiQueries,
+}) => {
+  const { strings } = useLanguageProvider()
+  const {
+    push,
+    updateQuery,
+    params: { dataViewCode },
+  } = useRouter()
+  const showToast = useToast({ style: 'negative' })
+
+  const { headerRow, tableRows, totalCount } = dataViewTable
+  const showDetailsForRow = (id: number) => push(`/data/${dataViewCode}/${id}`)
+
+  const sortByColumn = (
+    sortColumn: string | undefined,
+    columnTitle: string,
+    ascending: 'true' | 'false'
+  ) => {
+    if (!sortColumn) {
+      showToast({ text: strings.DATA_VIEW_COLUMN_NOT_SORTABLE.replace('%1', columnTitle) })
+      return
+    }
+    // Cycle through ascending/descending/not-sorted (default: id)
+    if (ascending === 'false') {
+      updateQuery({ sortBy: null })
+    } else if (!apiQueries.orderBy) updateQuery({ sortBy: sortColumn })
+    else
+      updateQuery({
+        sortBy: `${sortColumn}:desc`,
+      })
+  }
+
+  return (
+    <>
+      <Table stackable selectable sortable>
+        <Table.Header>
+          <Table.Row>
+            {headerRow.map(({ title, sortColumn }) => (
+              <Table.HeaderCell
+                key={title}
+                colSpan={1}
+                sorted={isSorted(sortColumn, apiQueries)}
+                onClick={() => sortByColumn(sortColumn, title, apiQueries?.ascending ?? 'true')}
+              >
+                {title}
+              </Table.HeaderCell>
+            ))}
+          </Table.Row>
+        </Table.Header>
+        <Table.Body>
+          {tableRows.map(({ id, rowValues }: { id: number; rowValues: any }) => (
+            <Table.Row
+              key={`row_${id}`}
+              className="clickable"
+              onClick={() => showDetailsForRow(id)}
+            >
+              {rowValues.map((value: any, index: number) => (
+                <Table.Cell key={`value_${index}`}>
+                  {getCellComponent(value, headerRow[index], id)}
+                </Table.Cell>
+              ))}
+            </Table.Row>
+          ))}
+        </Table.Body>
+      </Table>
+      <PaginationBar
+        totalCount={totalCount}
+        perPageText={strings.OUTCOMES_TABLE_PAGINATION_TEXT}
+        strings={strings}
+      />
+    </>
```

Comment:
All this is basically just the same as what was previously inside the parent component.